modules/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: CONCEPT
+[id="about-administrator-perspective_{context}"]
+= About the Administrator perspective in the web console
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective. The *Administrator* perspective enables you to view logging, metrics, and high-status information about the cluster. You can view the cluster inventory, capacity, general and specific utilization information, and the stream of important events, all of which help you to simplify planning and troubleshooting tasks. You can also view and manage a wide variety of settings that are of particular interest to the cluster administrators, such as cluster updates, cluster operators, CRDs, role bindings, and resource quotas. Both project administrators and cluster administrators can view the *Administrator* perspective.
+
+
```

Comment:
Not sure if this is anywhere on this draft, but I think it would be useful to add. It is from that Gdoc I sent.

> Cluster administrators can also launch an embedded command line terminal instance in the web console in OpenShift Container Platform 4.7 and later. The terminal instance is preinstalled with common CLI tools for interacting with the cluster, such as `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, and `kubectx`. It also has the context of the project you are working on and automatically logs you in using your credentials.